features/step_definitions/common_steps.rb

```diff
@@ -132,6 +132,10 @@
   page.all("." + button_link_class)[0].click
 end
 
+When /I click a #{MAYBE_VAR} link$/ do |link_name|
```

Comment:
Does the step above not work for this? It seems like it should?
ie the step defined as `When /I click #{MAYBE_VAR} ?(button|link)?$/ do |button_link_name, elem_type|`

Reply:
Annoyingly it gives an error if there's more than one matching element on the page. Our new step selects the first match.

Reply:
Ah yes, I made a step for this not too long ago - see https://github.com/alphagov/digitalmarketplace-functional-tests/blob/13b0d7b97bce822cb1943fc341c782b66f3901c5/features/step_definitions/common_steps.rb#L272

`I click the summary table Edit link for '#{field_to_edit}'` - should hopefully work for this summary table.  Maybe use that if we can rather than introduce this new step definition? 

(If it doesn't "just work" though then fine to add this).

Reply:
Oooh nice - I'll give that a go. Thanks Kev!